### PR TITLE
kinetica-langchain: New partner package

### PR DESCRIPTION
**Pull request by chadj2**
Currently, Kinetica has a number of [contributions](https://python.langchain.com/v0.1/docs/integrations/providers/kinetica/) to Langchain community:
* Chat Model
* Vector Store
* Document Loader
* Retriever

As we continue to expand our integrations we would like to consolidate these and future changes into a package. Please let me know if this is approved and I will follow up with the additional steps.

### CODE REVIEW

**Review by baskaryan — COMMENTED** [review]
could we mark all the langchain-community versions of integrations as `@deprecated(...)`

File: libs/partners/kinetica/docs/chat.ipynb
Comment (baskaryan) [inline libs/partners/kinetica/docs/chat.ipynb:1]: we'll still want to put these docs in the top level `docs/docs/integrations` directory

Comment (chadj2) [reply libs/partners/kinetica/docs/chat.ipynb:1]: I will follow up with the business on this. Do you want the deprecation decorations part of this PR?

Comment (chadj2) [reply libs/partners/kinetica/docs/chat.ipynb:1]: Management agreed that the community versions can be deprecated. Let me know what you need.

Comment (chadj2) [reply libs/partners/kinetica/docs/chat.ipynb:1]: We are ready to proceed. Are there any updates on this?

**Comment by efriis** [comment]
Hey @chadj2 ! Could you confirm if you discussed this with someone at LangChain per the [integration contributing docs](https://python.langchain.com/v0.2/docs/contributing/integrations/)? If so sorry I wasn't involved earlier, and I'm happy to reopen this!

If there are functionality changes in this, let's instead just modify the community integrations to add those features in a separate PR.

As a note, co-iterating on a PR is easier if you open it from a personal branch! https://python.langchain.com/v0.2/docs/contributing/faq/#how-do-i-allow-maintainers-to-edit-my-pr

**Comment by chadj2** [comment]
We have a number of community integrations that we want to consolidate into a partner package. Please let us know how we can proceed with this and if there are any additional requirements.

**Comment by efriis** [comment]
emailed!

**Pull request closed without merging**